### PR TITLE
Makes all CI checks pass (up to the master workflow)

### PR DESCRIPTION
**Pull request by ajsantander**
* Removes forking CI checks except for mainnet
* Minor bugfix on deployment script

### CODE REVIEW

**Review by justinjmoses — APPROVED** [review]
LGTM, though the `master` workflow is empty now.... not sure what if anything needs to be in there anymore: https://app.circleci.com/pipelines/github/Synthetixio/synthetix/5320/workflows/38c724dd-6aae-4d8e-a3cf-1616c4a6bdf0

**Comment by ajsantander** [comment]
> LGTM, though the `master` workflow is empty now.... not sure what if anything needs to be in there anymore: https://app.circleci.com/pipelines/github/Synthetixio/synthetix/5320/workflows/38c724dd-6aae-4d8e-a3cf-1616c4a6bdf0

Yep, it's a placeholder for now. We use to do check Etherscan verification there, but when we removed it, it was left empty.

File: test/prod/utils/ensureAccountHasBalance.js [hunk 1]
```diff
@@ -14,9 +13,11 @@ function getUser({ network, deploymentPath, user }) {
 }
 
 async function ensureAccountHasEther({ network, deploymentPath, amount, account }) {
+	console.log(gray(`  > Ensuring ${account} has Ether...`));
+
 	const fromAccount =
 		network === 'mainnet'
-			? knownMainnetWallet
+			? knownAccounts['mainnet'].find(a => a.name === 'binance').address
```
Comment (justinjmoses) [inline test/prod/utils/ensureAccountHasBalance.js:20]: [np] you could dedupe this at the top

**Review by justinjmoses — APPROVED** [review]
Nice one 💯 ! Just one nit.

File: test/prod/utils/ensureAccountHasBalance.js (hunk 1 above)
Comment (ajsantander) [reply test/prod/utils/ensureAccountHasBalance.js]: Fixed

**Pull request merged**